datastore/client.go

```diff
@@ -73,6 +73,17 @@ func (dc *datastoreClient) RunQuery(ctx context.Context, in *pb.RunQueryRequest,
 	return res, err
 }
 
+func (dc *datastoreClient) RunAggregationQuery(ctx context.Context, in *pb.RunAggregationQueryRequest, opts ...grpc.CallOption) (res *pb.RunAggregationQueryResponse, err error) {
```

Comment:
Do not add this code here. `RunAggregationQuery` is already in this package. If you'd like to add the trace in the `Client.RunAggregationQuery()` method, add that code here:

https://github.com/googleapis/google-cloud-go/blob/main/firestore/query.go#L1091

Reply:
Does `AggregationQuery.Get` exist for datastore? I only saw the firestore's one.

Reply:
Oops, that was the wrong link:
https://github.com/googleapis/google-cloud-go/blob/main/datastore/query.go#L629